client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -30,9 +28,12 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 	const { wpcomDomain } = useWooCommerceOnPlansEligibility( siteId );
 
 	const { get, save, update, countriesList } = useSiteSettings( siteId );
+	const countriesAsOptions = Object.entries( countriesList ).map( ( [ key, value ] ) => {
+		return { value: key, label: value };
+	} );
 
-	const handleCountryChange: ChangeEventHandler< HTMLInputElement > = ( event ) => {
-		update( WOOCOMMERCE_DEFAULT_COUNTRY, event.target.value );
+	const handleCountryChange = ( value: string ) => {
```

Comment:
The `onChange` callback for this function can sometimes be called with a `null` value, so you'll need to handle that here. Perhaps just with an early return?

Reply:
Null look like they happen when you click the x reset button, just set this to empty string, we'll be validating these values shortly.